Makefile

```diff
@@ -0,0 +1,9 @@
+# simple makefile to simplify repetitive build env management tasks
+
+test-coverage: python -m pytest test -m "not notebooks" --ignore=test/perf --ignore=test/install --cov=fairlearn --cov-report=xml --cov-report=html
+
+test-unit: python -m pytest ./test/unit
+
+test-perf: python -m pytest ./test/perf
```

Comment:
Does this work? To me that looks like the dependencies for target `test-perf` are the files "python", "-m", "pytest" and I'm not sure what `make` does when confronted with a directory as a dependency.

Reply:
I understood this to be an alias for a command, like it's used here: https://github.com/scikit-learn/scikit-learn/blob/master/Makefile

Reply:
@adrinjalali

Reply:
I'm testing this branch locally to check the makefile.